platform/android/scripts/metrics.sh

```diff
@@ -11,3 +11,7 @@ scripts/check_binary_size.js "platform/android/MapboxGLAndroidSDK/build/intermed
 
 # Track overall library size
 scripts/check_binary_size.js "platform/android/MapboxGLAndroidSDK/build/outputs/aar/MapboxGLAndroidSDK-release.aar" "Android AAR"
+
+if [[ $CIRCLE_BRANCH == binary-size-history ]]; then
```

Comment:
`binary-size-history` is temporary, to confirm that it's working. I will switch to `master` before merging.